src/Profiles/Shifting/DataPages/Base/ANTPLUS_ShiftingBaseMainDataPageMsg.h

```diff
@@ -0,0 +1,32 @@
+#ifndef ANTPLUS_SHIFTINGBASEMAINDATAPAGEMSG_h
+#define ANTPLUS_SHIFTINGBASEMAINDATAPAGEMSG_h
+
+#include <BaseClasses/ANTPLUS_BaseDataPageMsg.h>
+#include <Profiles/Shifting/ANTPLUS_ShiftingPrivateDefines.h>
+
+#include "ANT.h"
+
+class ShiftingBaseGenericMsg : public BaseDataPageMsg<BroadcastDataMsg> {
```

Comment:
I don't think we need the base datapage since there is only one datapage for this profile, can skip this and put it straight in TX

Reply:
What about manufacturer and product information ?

Reply:
Right, we need that to identify datapage number. But those datapages are from the common datapage set (those are pages 64 to 93) so they will go [here](https://github.com/cujomalainey/antplus-arduino/tree/master/src/CommonDataPages) and will inherit from the common datapage set.